WORKSPACE

```diff
@@ -12,6 +12,14 @@ load(
     "apple_rules_dependencies",
 )
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_python",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.2.0/rules_python-0.2.0.tar.gz",
+    sha256 = "778197e26c5fbeb07ac2a2c5ae405b30f6cb7ad1f5510ea6fdac03bded96cc6f",
+)
```

Comment:
Should this be added behind a `_maybe` in `rules_ios_dependencies()` [here](https://github.com/bazel-ios/rules_ios/blob/master/rules/repositories.bzl)?

Reply:
Yeah that's a good call, I'm not sure we _need_ this exact sha ATM

Reply:
@thiagohmcruz actually this is rules_ios's workspace, this is only used for rules_ios. I don't think this is needed anymore.

Reply:
I see! Well I'm always in favor of deleting unused code haha